lm-agent-snap/Makefile

```diff
@@ -0,0 +1,33 @@
+SHELL:=/bin/bash
+
+.PHONY: lint
+lint: ## Run the linters
+	tox -e lint
+
+.PHONY: type
+type: ## Run the type checker
+	tox -e type
+
+.PHONY: qa
+qa: lint type ## Run all quality assurance checks
+
+.PHONY: format
+type: ## Run the formatter
```

Comment:
```suggestion
format: ## Run the formatter
```

Reply:
fixed in [13c89e3](https://github.com/omnivector-solutions/license-manager/pull/389/commits/13c89e37e4fca4d4f8cd3de0090c4f227cc7a738)